projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -43,10 +43,13 @@ export const PROMPT_TYPE_LIST = [
 
 export type PromptTypeProp = ( typeof PROMPT_TYPE_LIST )[ number ];
 
-// Support backend prompts as a beta extension.
+// Enable backend prompts for beta sites + 10% of production sites.
+const blogId = parseInt( window?.Jetpack_Editor_Initial_State?.wpcomBlogId );
 export const areBackendPromptsEnabled: boolean =
 	window?.Jetpack_Editor_Initial_State?.available_blocks[ 'ai-assistant-backend-prompts' ]
-		?.available || false;
+		?.available ||
```

Comment:
shouldn't the first bool operation be `&&`? If not available (or not defined) should not allow for the evaluation to continue? 

Also, out of curiosity, why the last `|| false`?

Reply:
Just to close this, as we chatted in Slack:

- I need it to be an OR, since the user should use the backend prompts when it's a beta site as well as when the site ID was from the range specified; the evaluation will keep going until it gets to a truthy value or to the end of the expression
- The `|| false` was indeed not necessary :) removed on the last commit

Thanks for checking it!